invenio_app_ils/records_relations/api.py

```diff
@@ -421,12 +421,10 @@ def remove(self, previous_rec, next_rec, relation_type):
 class IlsRecordWithRelations(IlsRecord):
     """Add relations functionalities to records."""
 
-    @property
-    def relations(self):
+    def relations(self, sort_by=None):
```

Comment:
this is a breaking change in the class interface, did you test all the possible occurences?

Reply:
in addition: I think the sort should be somewhat defined per relation type (maybe config of relation types?) not as an argument of the function - we risk the list of arguments growing infinitely